.github/workflows/kong3.0_compatibility.yaml

```diff
@@ -0,0 +1,62 @@
+name: 'Kong 3.0 compatibility'
+
+on:
+  schedule:
+    - cron: '30 5 * * *'
+  workflow_dispatch: {}
+
+jobs:
+  integration-tests:
+    environment: "Configure ci"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        database-mode:
+          - 'dbless'
+          - 'postgres'
+        kong-edition:
+          - 'OSS'
+          - 'enterprise'
+    steps:
+    - name: setup golang
+      uses: actions/setup-go@v3
+      with:
+        go-version: '^1.19'
+
+    - name: cache go modules
+      uses: actions/cache@v3
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-build-codegen-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-build-codegen-
+
+    - name: checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: setup test kong image for OSS edition
+      if: ${{ matrix.kong-edition }} == 'OSS'
+      run: TEST_KONG_IMAGE=kong/kong;TEST_KONG_TAG=3.0.0-alpha.14-alpine
```

Comment:
Wow, I didn't know that you can set the env so easily. Have you tested it and it works? I thought that in Github Actions you have to do 

```
echo "{environment_variable_name}={value}" >> $GITHUB_ENV
```

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable.

Reply:
fixed. Actually I could not test it because GH did not let me run the action.